pkgs/tools/networking/linkchecker/default.nix

```diff
@@ -1,19 +1,5 @@
 { stdenv, lib, fetchFromGitHub, fetchpatch, python2, gettext }:
-let
-  # pin requests version until next release.
-  # see: https://github.com/linkcheck/linkchecker/issues/76
-  python2Packages = (python2.override {
-    packageOverrides = self: super: {   
-      requests = super.requests.overridePythonAttrs(oldAttrs: rec {
-        version = "2.14.2";
-        src = oldAttrs.src.override {
-          inherit version;
-          sha256 = "0lyi82a0ijs1m7k9w1mqwbmq1qjsac35fazx7xqyh8ws76xanx52";
-        };
-      });
-    };
-  }).pkgs;
-in
+
 python2Packages.buildPythonApplication rec {
```

Comment:
`python2.pkgs.buildPythonApplication` to fix eval

Reply:
Thanks @xeji, I now see that I made a mistake when testing it and the PR isn't actually valid so I'll close it.